src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
While only one version of `rospkg` can be installed that is not the case with `rospack`. Since there behavior must be the same that means that we also have to modify `ros-<distro>-rospack` despite the fact that it is independent of Debian.

Also the new env var needs to be set by some `ros-<distro>-?` package. While that package can depend on a new version number of `rospkg` there is no way to prevent users to update only `rospkg` but not that ros package. The result will be that on Ubuntu the user now has `/usr/share` in his RPP which will significantly increase the crawling time. This will also be the default case for distributions we have are not building anymore since `rospkg` will be deployed across all ROS distros.

Imo there are too many cons involved with this change. Therefore I still don't think that ROS should introduce yet another environment variable just to toggle this behavior necessary for Debian. But if the majority wants to go ahead with this I won't stand in the way.